components/chef-ui-library/src/charts/chef-radial-chart/chef-radial-chart.tsx

```diff
@@ -105,15 +110,17 @@ export class ChefRadialChart {
 
   @Method() updateDataPoints() {
     const dataPoints = Array.from(this.el.querySelectorAll('chef-data-point'));
-    this.dataPoints = groupBy((d) => d.secondary ? 'secondary' : 'primary', dataPoints);
+    const dataPointsWithValues = [];
+    dataPoints.forEach((d) => (Number(d.value) > 0) ? dataPointsWithValues.push(d) : null);
+    this.dataPoints = groupBy((d) => d.secondary ? 'secondary' : 'primary', dataPointsWithValues);
```

Comment:
Could shave off a line using `map`:

`const dataPointsWithValues = dataPoints.map(d => Number(d.value) > 0) ? d : null);`

Reply:
@scottopherson If I use map it will keep the null dataPoints in the array.

Reply:
@tarablack01 ah yea I see. Could go with `.filter` then:

`const dataPointsWithValues = dataPoints.filter(d => Number(d.value) > 0)`